correios/client.py

```diff
@@ -109,6 +110,8 @@ def _call(self, method_name, *args, **kwargs):
         method = getattr(self.sigep, method_name)
         try:
             return method(*args, **kwargs)
+        except ConnectTimeout:
+            raise ConnectTimeoutError
         except Fault as exc:
             if 'autenticacao' in str(exc):
```

Comment:
esse é o único/melhor jeito de ver se é um erro de autenticação? não tem nenhum atributo onde eu consiga pegar um código ou a mensagem dentro do objeto exc?

Reply:
Yes, executing some tests with SoapClient, to check authentications problems, I just get a Fault exception.
```python
Fault: A autenticacao de sigep falhou!
```

Reply:
a parte massa é que quando acontece erro de autenticação a API dos correios retorna código `500` ao invés de 40x :top: :100: :tada: